Gopkg.toml

```diff
@@ -10,38 +10,44 @@ required = [
   "k8s.io/gengo/args",
 ]
 
+[[override]]
```

Comment:
Want to add a comment as to why you added this? (Some weird dep thing, right?)

Reply:
yep, unsure if it's a local issue with my dep version, was hitting this: https://github.com/golang/dep/issues/1799